ntbs-service/deployments/README.md

```diff
@@ -0,0 +1,26 @@
+# SSL Certificates
+We use Let's Encrpyt for free certificates for the test environments
+*Note: we tried using an automated cert-manager approach as per: https://docs.microsoft.com/en-us/azure/aks/ingress-tls, but couldn't get it to work. Hence the manual appraoch*
+
+## Steps to create/renew certificate
+1. The environment will have to go down for a while - verify this being OK with its users
+1. Replace the application with a bare-bones nginx container, e.g. by
+  1. Replacing `spec.spec.containers[].image` with `"nginx:1.17.5" in the deployment yaml file
+  1. Running `kubeclt apply -f <deployment yaml file>`
+1. Wait for the nginx container to spin up - you should see a ngnix welcome page on the app environment's address
+1. Go inside the container:
+  1. Run `kubectl get pods` and note the pods name for your environment
+  1. `kubectl exec -it $podName -- bin/bash`
+1. In the container, follow the instructions from https://certbot.eff.org/lets-encrypt/debianbuster-nginx :
+  1. `apt-get update`
+  1. `apt-get install certbot python-certbot-nginx`
+  1. `certbot certonly --nginx`, supplying the environment's domain and team email address as prompted
```

Comment:
What does team email address refer to here?

Reply:
the DL here: https://swiki.softwire.com/display/PHE/NTBS+project+resources